### PR TITLE
[Update] Beginners Guide to Kubernetes Part 4

### PR DESCRIPTION
**Pull request by hzoppetti**
- reordered sections
- added note in replicaset
- updated language in deployment to come before replicaset

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit d72abcb28b8d0c7da592e16e6cfaa8de1c524e0b

https://deploy-preview-3220--nostalgic-ptolemy-b01ab8.netlify.com

File: docs/kubernetes/beginners-guide-to-kubernetes-part-4-controllers/index.md [hunk 1]
```diff
@@ -53,62 +53,61 @@ spec:
...
 
-In the above example, four of the Pods have already terminated, and one is in the process of terminating.
+For more information on Deployments, visit the [Kubernetes Deployments API documentation](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#deployment-v1-apps)
 
-For more information on ReplicaSets, view the [Kubernetes ReplicaSets API documentation](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#replicaset-v1-apps).
+## ReplicaSets
 
-## Deployments
+{{< note >}}
+Kubernetes now [recommends](https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/#when-to-use-a-replicaset) the use of Deployments instead of ReplicaSets. Deployments provide declarative updates to Pods among other features that allow you to define your application in the spec section. In this way, ReplicaSets have essentially become deprecated.
```
Comment (andystevensname) [inline docs/kubernetes/beginners-guide-to-kubernetes-part-4-controllers/index.md]: Comma after Pods, and comma after features.

File: docs/kubernetes/beginners-guide-to-kubernetes-part-4-controllers/index.md [hunk 2]
```diff
@@ -53,62 +53,61 @@ spec:
...
       containers:
       - name: apache-container
-        image: httpd
+        image: httpd:2.4.35
 {{</ file >}}
 
-There are three main things to note in this ReplicaSet. The first is the `apiVersion`, which is `apps/v1`. This differs from the previous examples, which were all `apiVersion: v1`, because ReplicaSets do not exist in the `v1` core. They instead reside in the `apps` group of `v1`. The second and third things to note are the `replicas` field and the `selector` field. The `replicas` field defines how many replica Pods you want to be running at any given time. The `selector` field defines which Pods, matched by their label, will be controlled by the ReplicaSet.
-
-To view your ReplicaSets, issue the `get replicasets` command:
+The only noticeable difference between this Deployment and the example given in the [ReplicaSet](#replicasets) section is the `kind`. In this example we have chosen to initially install Apache 2.4.35. If you wanted to update that image to Apache 2.4.38, you would issue the following command:
```
Comment (andystevensname) [inline docs/kubernetes/beginners-guide-to-kubernetes-part-4-controllers/index.md]: "As you will see, the only noticeable difference between a Deployment's manifest and that of a ReplicaSet is the `kind`." Just trying to point ahead to ReplicaSets and make it more specific.

File: docs/kubernetes/beginners-guide-to-kubernetes-part-4-controllers/index.md (hunk 1 above)
Comment (hzoppetti) [reply docs/kubernetes/beginners-guide-to-kubernetes-part-4-controllers/index.md]: updated

File: docs/kubernetes/beginners-guide-to-kubernetes-part-4-controllers/index.md (hunk 2 above)
Comment (hzoppetti) [reply docs/kubernetes/beginners-guide-to-kubernetes-part-4-controllers/index.md]: updated

**Pull request merged**